identity/app/views/completeConsents.scala.html

```diff
@@ -22,12 +22,13 @@
     <section class="identity-forms-message">
         <h1 class="identity-title">Thank you! We've got your preferences</h1>
         <div class="identity-forms-message__body">
-            <p>You can change these anytime by going to My account > <a class="u-underline" href="@idUrlBuilder.buildUrl(controllers.editprofile.routes.EditProfileController.displayEmailPrefsForm(false, None).url, idRequest)">Email
-                Preferences</a>.</p>
+            <p>You can change these anytime by going to My account > <a class="u-underline" data-link-name="complete-consents : to-email-prefs" href="@idUrlBuilder.buildUrl(controllers.editprofile.routes.EditProfileController.displayEmailPrefsForm(false, None).url, idRequest)">
+                Email Preferences
+            </a>.</p>
             <br/>
         </div>
         @if(!emailPrefsForm.globalError) {
-            <form novalidate action="@buildIdentityUrl("email-prefs")" role="main" method="post" class="js-manage-account__ajaxForm">
```

Comment:
I don't know what this was previously needed for, so can't comment on its removal

Reply:
it isn't (there's no buttons inside) so I removed it just in case to keep more control over the urls listed in the page